airflow/www/templates/airflow/version.html

```diff
@@ -0,0 +1,23 @@
+{% extends "airflow/master.html" %}
+
+{% block title %}
+    {{ title }}
+{% endblock %}
+
+{% block body %}
+    {{ super() }}
+    <h2>{{ title }}</h2>
+    {% if airflow_version %}
+	    <h4>Version : <a href="https://pypi.python.org/pypi/airflow/{{ airflow_version }}"> {{ airflow_version }}</a></h4>
+    {% else %}
+	    <h4>Version : Not Available</a></h4>
+    {% endif %}
+
+    {% if git_proj_uri and git_hash %}
+	    <h4>Git Commit Tree : <a href="https://{{ git_proj_uri }}/commits/{{ git_hash }}"> {{ git_hash }}</a></h4>
+    {% else %}
+	    <h4>Git Commit Tree : Not Available</a></h4>
+    {% endif %}
+	<hr>
```

Comment:
Good catch. Damn tabs snuck in multiple places .. have fixed them all in this file.